inc/Engine/Optimization/Minify/CSS/AbstractCSSOptimization.php

```diff
@@ -121,14 +121,6 @@ protected function is_minify_excluded_file( array $tag ) {
 			return true;
 		}
 
-		if ( false !== strpos( $tag[0], 'media=' ) && ! preg_match( '/media=["\'](?:\s*|[^"\']*?\b(all|screen)\b[^"\']*?)["\']/i', $tag[0] ) ) {
```

Comment:
keep this line to avoid processing stylesheets for `print` or `speech`

Reply:
Ok and what about this
```
if ( false !== strpos( $tag[0], 'only screen and' ) ) {
			return true;
		}
```